src/community/resources-and-tools/index.md

```diff
@@ -100,13 +100,12 @@ A Vue 3 component library for GOV.UK Frontend
 
 ## Write code
 
-You can download GOV.UK Frontend Nunjucks macro snippets for:
+[GOV.UK Design System snippets for Nova](https://extensions.panic.com/extensions/ca/ca.GOVUKDesignSystemSnippets/).
 
-- [Nova](https://extensions.panic.com/extensions/ca/ca.GOVUKDesignSystemSnippets/)
-- [Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=simonwhatley.govuk-design-system-snippets)
+[GOV.UK Design System snippets for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=simonwhatley.govuk-design-system-snippets).
```

Comment:
These 2 are not approved in spreadsheet (yet) J1122 and J1123

Reply:
I've gone back and forth on if these should include 'Nunjucks' ie: 'GOV.UK Design System _Nunjucks_ snippets' but actually I think this is fine. If this gets flagged as unclear we'll change it.